main/admin/index.php

```diff
@@ -396,6 +396,13 @@
         'url' => api_get_path(WEB_CODE_PATH).'calendar/agenda_js.php?type=admin',
         'label' => get_lang('GlobalAgenda'),
     ];
+
+    $items[] = [
+        'class' => 'item-agenda-reminders',
+        'url' => api_get_path(WEB_CODE_PATH).'admin/import_course_agenda_reminders.php',
+        'label' => get_lang('AgendaReminders'),
+    ];
+
```

Comment:
Shouldn't this link appear *only* if api_get_configuration_value('agenda_reminders') is true ?